common/infrastructure/github/labels.tf

```diff
@@ -0,0 +1,10 @@
+data "github_repository" "ludicrum" {
```

Comment:
Here you're creating the repo again. This will introduce conflicts. You can reference the repo from the resource already created in the file with the same name.